frontend/src/components/Sidebar/MapComponents/HeatMap.tsx

```diff
@@ -1,21 +1,27 @@
 // SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
 // SPDX-License-Identifier: Apache-2.0
 
-import React, {useState, useEffect, useRef, useMemo, useCallback, useLayoutEffect} from 'react';
+// React imports
+import React, {useCallback, useEffect, useLayoutEffect, useMemo, useRef, useState} from 'react';
+
+// Third-party
 import * as am5 from '@amcharts/amcharts5';
 import * as am5map from '@amcharts/amcharts5/map';
-import {Feature, GeoJSON, GeoJsonProperties} from 'geojson';
-import svgZoomResetURL from '../../../../assets/svg/zoom_out_map_white_24dp.svg?url';
-import svgZoomInURL from '../../../../assets/svg/zoom_in_white_24dp.svg?url';
-import svgZoomOutURL from '../../../../assets/svg/zoom_out_white_24dp.svg?url';
```

Comment:
I think we can delete these assets from the project.

Reply:
Deleted.
https://github.com/DLR-SC/ESID/pull/384/commits/59b91ba55b9145fdea75d8cbaa28e3e8bfc240e2